examples/resources/infoblox_a_record.tf

```diff
@@ -26,3 +26,13 @@ resource "infoblox_a_record" "rec3" {
   ttl = 0 // 0 = disable caching
   ext_attrs = jsonencode({})
 }
+
+// name parameter recommended to use with lower() function
+resource "infoblox_a_record" "rec4" {
+  fqdn = lower("NEWRECORD2.example2.org") // if you want to use uppercase letters, please use this 'lower' function
```

Comment:
You should not use a separate example resource to demonstrate this. Please, use one of the existing resources (rec1, rec2, rec3).